File: costVariance.cpp
// costVariance.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <iostream>

float bcwp()
{
	std::cout << "Enter the cumalative BCWP: ";
	float bcwp;
	std::cin >> bcwp;
	return bcwp;
}


float acwp()
{
	std::cout << "Enter the cumalative ACWP: ";
	float acwp;
	std::cin >> acwp;
	return acwp;
}

void writeVariance(float result)
{
	std::cout << "The current project CPI is:" << result << std::endl;
}

float main()
{
	float cpi = bcwp() / acwp();
	writeVariance(cpi);
    return 0;
}

File: costVariance.py
# -*- coding: utf-8 -*-
"""
Created on Fri Jun  1 07:55:49 2018

@author: Scott Warnock
"""

# costVariance.py

def main():
    bcws = eval(input("Enter the cumalative planned value: "))
    bcwp = eval(input("Enter the cumalative earned value: "))
    acwp = eval(input("Enter the cumalative actual cost to date: "))
    
    cpi = bcwp / acwp
    
    print("Your cumalative CPI is: ", round(cpi,2))
    
    if cpi > 1:
        print("Your project is currently under costed.")
    elif cpi < 1:
        print("Your project is currently over costed.")
    else:
        print("Your project costs are inline with you earned value.")
    
main()